src/module.js

```diff
@@ -951,6 +957,8 @@ export default class DailyIframe extends EventEmitter {
           }
         }
       }
+      properties.showLocalVideo = this._showLocalVideo;
```

Comment:
vice-versa, no? don't we want to sync our initial local state (`this._showLocalVideo`) to what the user is setting when joining the call (`properties.showLocalVideo`), _if_ they're setting this value at join time?

Reply:
or am i missing something? it looks like, as implement here, if the user were to join a call with `properties.showLocalVideo = false`, then `showLocalVideo()` would not report the right value